registry/storage/driver/s3-aws/s3.go

```diff
@@ -230,6 +243,8 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		fmt.Sprint(rootDirectory),
 		storageClass,
 		fmt.Sprint(userAgent),
+		fmt.Sprint(backend),
```

Comment:
@kmala you don't need `fmt.Sprintf` for this line or the above, or the `fmt.Sprint(rootDirectory)` line a few lines above that